shard.yml

```diff
@@ -4,7 +4,7 @@ version: 0.2.0
 authors:
   - Vitalii Elenhaupt <velenhaupt@gmail.com>
 
-crystal: ">= 0.35.0"
+crystal: 1.8.1
```

Comment:
can we do a minimal crystal req here? i.e. `>= 1.0` ?

Reply:
> can we do a minimal crystal req here? i.e. `>= 1.0` ?

absolutely :)